src/overlays/actors/ovl_Obj_Tokei_Turret/z_obj_tokei_turret.h

```diff
@@ -6,10 +6,11 @@
 struct ObjTokeiTurret;
 
 typedef struct ObjTokeiTurret {
-    /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x18];
+    /* 0x000 */ DynaPolyActor dyna;
 } ObjTokeiTurret; // size = 0x15C
 
 extern const ActorInit Obj_Tokei_Turret_InitVars;
 
+#define OBJTOKEI_DISPLAY_STATE(thisx) ((thisx)->params & 3)
```

Comment:
I don't think `DISPLAY_STATE` is quite right for this anyway. How about `TIER_TYPE` or something?